package.json

```diff
@@ -97,33 +97,33 @@
     "@nuxtjs/eslint-module": "^3.0.2",
     "@nuxtjs/storybook": "^4.3.0",
     "@nuxtjs/style-resources": "^1.0.0",
-    "@pinia/testing": "^0.0.9",
+    "@pinia/testing": "^0.0.11",
     "@playwright/test": "^1.16.2",
     "@testing-library/jest-dom": "^5.16.1",
-    "@testing-library/user-event": "^13.2.1",
-    "@testing-library/vue": "^5.8.2",
+    "@testing-library/user-event": "^14.0.4",
+    "@testing-library/vue": "^6.5.1",
     "@types/express-useragent": "^1.0.2",
-    "@types/jest": "^26.0.22",
+    "@types/jest": "^27.4.1",
     "@types/module-alias": "^2.0.1",
     "@types/uuid": "^8.3.4",
     "@typescript-eslint/eslint-plugin": "^5.15.0",
     "@typescript-eslint/parser": "^5.15.0",
     "@vue/runtime-dom": "^3.2.31",
-    "@vue/test-utils": "^1.1.3",
+    "@vue/test-utils": "^2.0.0-rc.17",
```

Comment:
We can't update `@vue/test-utils` to v2.x because it [targets Vue 3 ](https://vue-test-utils.vuejs.org/) and Nuxt depends on Vue 2 currently.